internal/controller/persistentvolume/persistentvolume.go

```diff
@@ -82,7 +82,13 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	}
 
 	// Watch for changes to PersistentVolumes
-	err = c.Watch(source.Kind(mgr.GetCache(), &corev1.PersistentVolume{}), &handler.EnqueueRequestForObject{})
+	err = c.Watch(source.Kind(
+		mgr.GetCache(),
+		&corev1.PersistentVolume{},
+		handler.TypedEnqueueRequestForOwner[*corev1.PersistentVolume](
```

Comment:
should be `TypedEnqueueRequestForObject[*corev1.PersistentVolume]` ?

Reply:
yes thats looks good. updated now.